src/test/java/com/microsoft/sqlserver/UnitStatement/CallableMixed.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.UnitStatement;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.util.RandomUtil;
+
+@RunWith(JUnitPlatform.class)
+public class CallableMixed extends AbstractTest {
+    Connection connection1 = null;
+    Statement Statement151 = null;
```

Comment:
Here I found non standard naming conventions. Instance Variables should start with lowercase. Generally identified as **camelCase** / **lowerCamelCase**. Look difference between connection1 and Stmatement151 / Driver1. Please change all instance variables with appropriate camelCase naming convention.

PS: Generally only static / final variables  are in CAPS otherwise camelCase is pretty standard.